kolibri/core/assets/src/views/AppBody.vue

```diff
@@ -51,16 +53,24 @@
       ...mapState({
         loading: state => state.core.loading,
         blockDoubleClicks: state => state.core.blockDoubleClicks,
-        error: state => state.core.error,
       }),
-      padding() {
-        return this.windowIsSmall ? 16 : 32;
+      isMobile() {
```

Comment:
We can get rid of this computed prop by just referencing `windowIsSmall` instead.